crates/sui-tool/src/commands.rs

```diff
@@ -177,21 +177,26 @@ pub enum ToolCommand {
         max_content_length: usize,
     },
 
-    /// Download all packages to the local filesystem from an indexer database. Each package gets
-    /// its own sub-directory, named for its ID on-chain, containing two metadata files
-    /// (linkage.json and origins.json) as well as a file for every module it contains. Each module
-    /// file is named for its module name, with a .mv suffix, and contains Move bytecode (suitable
-    /// for passing into a disassembler).
+    /// Download all packages to the local filesystem from a GraphQL service. Each package gets its
+    /// own sub-directory, named for its ID on-chain and version containing two metadata files
```

Comment:
```suggestion
    /// own sub-directory, named for its ID on chain and version containing two metadata files
```
For consistency...named for its ID on chain, but could also be named for its on-chain ID.